x-pack/solutions/security/plugins/security_solution/public/asset_inventory/components/empty_state.tsx

```diff
@@ -11,17 +11,9 @@ import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { css } from '@emotion/react';
 import illustration from '../../common/images/illustration_product_no_results_magnifying_glass.svg';
+import { DOCS_URL, TEST_SUBJ_EMPTY_STATE } from '../constants';
 
-const ASSET_INVENTORY_DOCS_URL = 'https://ela.st/asset-inventory';
-const EMPTY_STATE_TEST_SUBJ = 'assetInventory:empty-state';
-
-export const EmptyState = ({
-  onResetFilters,
-  docsUrl = ASSET_INVENTORY_DOCS_URL,
-}: {
-  onResetFilters: () => void;
-  docsUrl?: string;
-}) => {
+export const EmptyState = ({ onResetFilters }: { onResetFilters: () => void }) => {
```

Comment:
EmptyState is generic and i suggest to save this type of naming to generic components. this one is a wrapper of asset inventory around eui empty state components. i suggest to name file and component as `asset inventory empty state`